### PR TITLE
Create Checklinks action

### PR DESCRIPTION
**Pull request by larryclaman**
This PR implements a Github actions workflow to automatically validate links within the WTH repo.
At this time, it only validates 001IntroToAKS, but it could be trivially extended to validate the other challenges.

### CODE REVIEW

File: .github/workflows/checklinks.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#        uses: lycheeverse/lychee-action@v1.0.8
+#        with:
+#          args: --no-progress **/*.md
+
+        # https://github.com/gaurav-nelson/github-action-markdown-link-check
+      - name: Markdown Links Check
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'
+          max-depth: 3
```
Comment (plzm) [inline .github/workflows/checklinks.yml:27]: Why the max-depth limit to 3 folders?

Comment (plzm) [reply .github/workflows/checklinks.yml:27]: LGTM otherwise.

**Review by plzm — APPROVED** [review]
Approving since my question about why three levels is non-blocking.

**Comment by larryclaman** [comment]
> Why the max-depth limit to 3 folders?
Good question.... and I don't recall the exact reason I set this unfortunately.  I think it may have had to do with avoiding the traversal of the source code.

**Comment by plzm** [comment]
Adding @jrzyshr as second reviewer. Peter, this looks OK to me, the path exclusions in the action limit its blast radius. I already approved it but needs a second.

**Review by Infinitecupcakes — APPROVED** [review]
approve per plzm

**Pull request merged**